### PR TITLE
[V2 2623] update cli for new btr

### PR DESCRIPTION
**Pull request by bpeters**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

It updates the `publish` and `update` commands to interface with the new BTR endpoint changes.
It adds the `release` command to interface with the new BTR route.
It adds the `rollback` command to interface with the new BTR route.

* **What is the new behavior (if this is a feature change)?**

New blocks need to be published to create a version which initializes the block and sets it for staging. The update is used now to update the bundle for the block and toggle isPublic.
The release is used to set the block for actual consumption to store users.
The rollback is used to move a recent release back to staging and set a previous release active.

Major version is now supported using a git branch solution for managing code bases and .element-block files.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

Yes, it must use the latest BTR endpoints.

* **Other information**:

### CODE REVIEW

**Comment by bpeters** [comment]
@santios-volusion ready for review

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@volusion/element-cli",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (santios-volusion) [inline package.json]: @bpeters We use a tool called `np` (check packages.json) that ask us what do we want to do with the version before releasing, and releases the package to npm. Maybe we should keep relying on that.

Comment (bpeters) [reply package.json]: oh okay, sorry I needed to bump locally to get BTR to work :) I'll remove from PR

Comment (santios-volusion) [reply package.json]: @bpeters Thanks!

**Pull request merged**